modules/memfault-firmware-sdk/memfault_integration.c

```diff
@@ -35,7 +35,7 @@ LOG_MODULE_REGISTER(memfault_ncs, CONFIG_MEMFAULT_NCS_LOG_LEVEL);
 
 /* Project key check */
 BUILD_ASSERT(sizeof(CONFIG_MEMFAULT_NCS_PROJECT_KEY) > 1,
-	"Memfault Project Key not configured. Please visit " MEMFAULT_URL);
+	"Memfault Project Key not configured. Please visit " MEMFAULT_URL " ");
```

Comment:
not blocking over this, but are you expecting to add spaces to every source code that might print a url on a build assert if they don't have an extra space ?

and what about upstream source code.

also remember that without a code comment here, this is likely to be removed by someone cleaning up the code.

Isn't it possible to handle this in the VSCode extension ?

Reply:
I believe VS Code extension cant do much about this because this log appears in the terminal (which is an external application). Correct me if I am wrong @trond-snekvik

Reply:
We don't control the link provider in the terminal, so we won't be able to do anything about this, I'm afraid. Quotation marks are technically legal in URLs, and although vscode removes wrapping quotation marks, having one at the end doesn't  work.